docs/reference/pip_install.rst

```diff
@@ -409,6 +392,39 @@ Installing a package from a wheel does not invoke the build system at all.
 .. _PyPI: http://pypi.python.org/pypi/
 .. _setuptools extras: http://packages.python.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies
 
+Controlling setup_requires()
+++++++++++++++++++++++++++++
+
+Setuptools offers the ``setup_requires`` `setup() keyword
+<http://pythonhosted.org/setuptools/setuptools.html#new-and-changed-setup-keywords>`_
+for specifying dependencies that need to be present in order for the `setup.py`
+script to run.  Internally, Setuptools uses ``easy_install`` to fulfill these
+dependencies.
+
+pip has no way to control how these dependencies are located.  None of the
+Package Index Options have an effect.
+
+The solution is to configure a "system" or "personal" `Distutils configuration
+file
+<http://docs.python.org/2/install/index.html#distutils-configuration-files>`_ to
+manage the fulfillment.
+
+For example, to have the dependency located at an alternate index, add this:
```

Comment:
This prose was just moved. Its not setup.cfg specific - e.g. that can go into ~/.pydistutils.cfg. The link given in the prose lists the files it can be put into.